llama-index-integrations/agent/llama-index-agent-openai/llama_index/agent/openai/step.py

```diff
@@ -446,6 +549,12 @@ def _run_step(
         openai_tools = [tool.metadata.to_openai_tool() for tool in tools]
 
         llm_chat_kwargs = self._get_llm_chat_kwargs(task, openai_tools, tool_choice)
+        for tool in tools:  # self.agent_worker._get_tools("test"):
```

Comment:
Are these debug prints? Can we remove them?

Reply:
Indeed, removing them in the next commit